### ISSUE TITLE
Long values does not convert

### ISSUE THREAD

**Issue by mlvn23** [uniVocity/univocity-parsers#30]
If I change the data type to Integer, it would convert. It looks like you missed it in AnnotationHelper. I tried using the @Convert annotation, but it didn't work because it's looking for a construction with a String... arguments, but LongConversion has none.

**Comment by jbax** [comment]
Fixed in this commit
https://github.com/uniVocity/univocity-parsers/commit/7ac2788c150b20a5eb51cc51ae433dc4d944db9c

I released SNAPHOST version 1.5.5. WIll release final version and merge into 1.6.0 once documentation is ready.

**Comment by mlvn23** [comment]
Thanks!

**Comment by jbax** [comment]
Merged into 1.6.0. Please update to 1.6.0-SNAPSHOT
